include/engine_x64_nts.h

```diff
@@ -368,23 +384,20 @@ typedef struct _zend_class_constant {
 typedef struct _zend_internal_arg_info {
     const char *name;
     zend_type type;
-    zend_uchar pass_by_reference;
-    zend_bool is_variadic;
+    const char *default_value;
```

Comment:
Change is not compatible between versions, unfortunately. Need to have header files per PHP version, platform and TS mode

Reply:
my approach was that there will be always only one PHP version supported. I wanted to do minimal effort and maintenance. No problem with multiple version support though, but the maintenance can become more tricky with this